test/test.js

```diff
@@ -126,6 +126,24 @@ test('linting of incorrect Markdown file fails with absolute path', async t => {
   }
 });
 
+test('linting of unreadable Markdown file fails', async t => {
+  const unreadablePath = '../unreadable.test.md';
+  try {
+    fs.writeFileSync(unreadablePath, '', {mode: 0o222});
+
+    try {
```

Comment:
Not sure we need the nested try? I think the outer can have a catch and a finally?